src/app/plugins/platform/Docker/plugins/cms/Drupal8/index.ts

```diff
@@ -273,50 +278,27 @@ class Drupal8 extends Generator {
 
   writing() {
     const needsMemcached = this.options.plugins.cache === 'Memcache';
-    const sharedDependencies = needsMemcached
-      ? ['libmemcached-dev', 'zlib-dev', 'libevent-dev']
-      : [];
-
-    const sharedPeclPackages = needsMemcached ? ['memcached'] : [];
+    const sharedDependencies = needsMemcached ? [memcached] : [];
+    sharedDependencies.push(opcache);
 
     const drupalDockerfile = createPHPDockerfile({
       from: { image: 'drupal', tag: this.latestDrupalTag },
-      buildDeps: sharedDependencies,
-      peclPackages: sharedPeclPackages,
-      xdebug: true,
+      dependencies: sharedDependencies,
     });
 
-    // "borrowed" from library/drupal's Dockerfile
-    const drupalDependencies = [
-      'coreutils',
-      'freetype-dev',
-      'libjpeg-turbo-dev',
-      'postgresql-dev',
-      'libzip-dev',
-    ];
+    const drupalDependencies = [gd, pdo, zip];
```

Comment:
Are these the platform dependencies for the composer file?

Reply:
These are the dependencies that the Drupal image uses on the Docker Hub - see [this Dockerfile](https://github.com/docker-library/drupal/blob/d560abe7847e839f6a130c37605966e160f3ac88/8.7/fpm-alpine/Dockerfile#L8).